README.md

```diff
@@ -38,5 +52,28 @@ In order to configure the client using the `hazelcast-client.yml` file, place th
 
 Configuration entries from `hazelcast-client.yml` override all `quarkus.hazelcast-client.*` entries.
 
+## Testing
+
+To make testing simple, Quarkus provides the `HazelcastServerTestResource` which automatically launches an embedded Hazelcast instance with defaults settings and manages its lifecycle:
```

Comment:
`Quarkus provides` --> is this quarkus provided?

Reply:
No, it's us :) good catch